dev/fattest.databases/src/componenttest/topology/database/container/DatabaseContainerType.java

```diff
@@ -12,42 +12,49 @@
 
 import static org.junit.Assert.fail;
 
+import java.lang.reflect.Constructor;
+
 import org.testcontainers.containers.JdbcDatabaseContainer;
 import org.testcontainers.containers.output.OutputFrame;
 
+import com.ibm.websphere.simplicity.config.DataSourceProperties;
 import com.ibm.websphere.simplicity.log.Log;
 
 /**
- * This is a current list of database test-containers that are in the database rotation.
+ * This is a current list of database testcontainers that are in the database rotation.
  */
 @SuppressWarnings("rawtypes")
 public enum DatabaseContainerType {
-    DB2("jcc.jar", "org.testcontainers.containers.", "Db2Container"),
-    Derby("derby.jar", "componenttest.topology.database.container.", "DerbyNoopContainer"),
-    DerbyClient("derbyclient.jar", "componenttest.topology.database.container.", "DerbyClientNoopContainer"),
-    Oracle("ojdbc8_g.jar", "org.testcontainers.containers.", "OracleContainer"),
-    Postgres("postgresql.jar", "org.testcontainers.containers.", "PostgreSQLContainer"),
-    SQLServer("mssql-jdbc.jar", "componenttest.topology.database.container.", "SQLServerContainer");
+    DB2("jcc.jar", "org.testcontainers.containers.", "Db2Container", "Properties_db2_jcc"),
```

Comment:
Where does org.testcontainers.containers exist?

Reply:
That is part of the testcontainers library that we pull in here:
https://github.com/OpenLiberty/open-liberty/blob/integration/dev/fattest.databases/bnd.bnd#L33-L35

Reply:
Thanks!